### DISCUSSION TITLE
Are the branch names being merged available as a variable for the Mergetool command?

### DISCUSSION THREAD

**Discussion by asfarley** [gitextensions/gitextensions#8480] [General] [answered]
I would like to pass the branch names being merged to Beyond Compare, because sometimes I don't remember which is REMOTE and which is LOCAL. 

Are the branch names available as variables, simliar to $LOCAL and $REMOTE?

**Comment by gerhardol** [comment, marked as answer]
No, when filenames are to be compared only the commits are available.
You select commits rather than branches, especially for diff.
For merge/rebase the current checked out branch could be included somehow, but the target branch could only be guessed. Not straight forward to display something like this.
